dist/panel-triggers/module.js

```diff
@@ -9784,7 +9784,13 @@ var AlertCard = /** @class */ (function (_super) {
         };
         _this.onLinkIconClick = function (event, url) {
             event.stopPropagation();
-            window.top.location.href = url;
+            // Add 'foldTabRow' query param so that Grafana tab row panel keeps folded
+            var urlBase = url.substring(0, url.indexOf('?') > -1 ? url.indexOf('?') : url.length);
+            var urlQuery = url.substring(url.indexOf('?') > -1 ? url.indexOf('?') : url.length);
+            var queryObj = _this.parseParamsStringToObject(urlQuery);
+            queryObj['foldTabRow'] = 'true';
```

Comment:
Do we have use case for multiple tab rows in single dashboard? If so, simple boolean won't be sufficent.

Reply:
To support multiple tab rows the multiple parameters should be supported and maybe for current use value "all" to indicate it applies to all tab rows. How does that sound?

Reply:
Ok, lets use keyword "all" now for starters